### PR TITLE
state-machine: prep for a minor release

### PR DESCRIPTION
**Pull request by tzemanovic**

### CODE REVIEW

File: proptest-state-machine/src/strategy.rs [hunk 1]
```diff
@@ -567,7 +567,7 @@ mod test {
     ///
     /// This constant can be determined from the test
     /// `number_of_sequential_value_tree_simplifications`.
-    const SIMPLIFICATIONS: usize = 699;
+    const SIMPLIFICATIONS: usize = 32;
```
Comment (rexmas) [inline proptest-state-machine/src/strategy.rs:570]: for clarity why this change?

Comment (tzemanovic) [reply proptest-state-machine/src/strategy.rs:570]: I missed this change in #392 that fixed a bug in shrinking, it was actually failing on that branch

**Review by rexmas — APPROVED** [review]
👍
